app/services/marketplace_service/api/marketplaces.rb

```diff
@@ -116,6 +116,13 @@ def set_locales(community, locales)
       community_name = community.name(default_locale)
       locales.each { |locale| Helper.first_or_create_community_customization!(community, community_name, locale) }
 
+      # Replace removed locale with default for users of marketplace
+      removed_locales = community.locales - locales
+      to_locale = removed_locales.include?(default_locale) ? locales.first : default_locale
```

Comment:
If I read this piece of code right, the `default_locale` here is the old default locale before the update. The new default locale will be the `locales.first`. So would it make sense to set the `to_locale` to `locales.first` in every case?